engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/OptimizedEvaluationContext.scala

```diff
@@ -32,13 +37,36 @@ class EvaluationContextPreparer(classLoader: ClassLoader,
     expressionImports.foreach(registerImport)
   }
 
+  private val mockPackageBlackList: Set[Regex] = Set("^(java.lang.System).*$".r, "^(java.math).*$".r)
+
+  private def blockBlacklisted(targetObject: scala.Any, methodName: String): Unit = {
+    //todo: get blacklist from ExpressionConfig
+
+    val classFullName = targetObject.asInstanceOf[Class[_]].getName
+    mockPackageBlackList.find(blackListRegex =>
```

Comment:
i'd move this logic to spelExpressionBlacklist - then you could add some unit tests and this logic would be separeted